.github/workflows/docker-publish.yml

```
@@ -0,0 +1,121 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  #schedule:
+  #  - cron: '16 5 * * *'
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  reviewdog-hadolint:
+
+    name: Lint Dockerfile
+    runs-on: ubuntu-latest
```

```suggestion
jobs:
  reviewdog-hadolint:

    name: Lint Dockerfile
    runs-on: ubuntu-latest
    permissions:
      contents: read
      pull-requests: write
```

pull-requests write permission need only reviewdog